server/src/main.rs

```diff
@@ -21,7 +21,12 @@ mod tables;
 mod types;
 mod storage;
 
-use std::sync::Arc;
+
+use std::{fs::File, sync::Arc, fmt};
```

Comment:
As mentioned in the comment to `catalog.rs`  we should group import manually as std -> external crates -> internal crates. tracing, tokio and datafusion should therefore be in one group.

Reply:
Fixed